contracts/compound/ComptrollerPrudentiaCapsExt.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { DiamondExtension } from "../midas/DiamondExtension.sol";
+import { ComptrollerErrorReporter } from "../compound/ErrorReporter.sol";
+import { CTokenInterface, CTokenExtensionInterface, CErc20Interface } from "./CTokenInterfaces.sol";
+import { ComptrollerV4Storage } from "./ComptrollerStorage.sol";
+
+/**
+ * @title ComptrollerPrudentiaCapsExt
+ * @author Tyler Loewen (TRILEZ SOFTWARE INC. dba. Adrastia)
+ * @notice A diamond extension that allows the Comptroller to use Adrastia Prudentia to control supply and borrow caps.
+ */
+contract ComptrollerPrudentiaCapsExt is DiamondExtension, ComptrollerV4Storage, ComptrollerErrorReporter {
```

Comment:
looks like this extension does not need to inherit the `ComptrollerErrorReporter`

Reply:
Good catch! Fixed.